### ISSUE TITLE
Error when calling readHeader method

### ISSUE THREAD

**Issue by cj848** [usefulness/webp-imageio#111]
java.lang.UnsatisfiedLinkError: 'int com.luciad.imageio.webp.WebP.getInfo(byte[], int, int, int[])'
        at com.luciad.imageio.webp.WebP.getInfo(Native Method)
        at com.luciad.imageio.webp.WebPWrapper.getInfo(WebPWrapper.kt:49)
        at com.luciad.imageio.webp.WebPReader.readHeader(WebPReader.kt:52)
        at com.luciad.imageio.webp.WebPReader.getWidth(WebPReader.kt:95)

An error occurred while calling the above stacktrace to read the header of the webp file. It continues to occur from 0.3.1 to 0.4.0.

**Comment by mateuszkwiecinski** [comment]
Please do try [0.5.0-SNAPSHOT](https://github.com/usefulness/webp-imageio/packages/1476285) available via 
```kotlin
repositories {
    maven {
        url = uri("https://maven.pkg.github.com/usefulness/webp-imageio")
        credentials {
            username = project.findProperty("gpr.user") ?: System.getenv("USERNAME")
            password = project.findProperty("gpr.key") ?: System.getenv("TOKEN")
        }
   }
}
```
Github Packages documentation: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry#using-a-published-package

If things work in an expected way now, I'll publish a normal release

**Comment by cj848** [comment]
i don't have your gpr.user, key. i think your commit is work.

**Comment by mateuszkwiecinski** [comment]
these two are your own credentials, it's your username and your [Personal Access Token](https://github.com/settings/tokens)
```
repositories {
    maven {
        url = uri("https://maven.pkg.github.com/usefulness/webp-imageio")
        credentials {
            username = "cj848"
            password = "your_personal_access_token"
        }
   }
}
```
The documentation I [linked](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry#authenticating-to-github-packages) above shares more details on how to authenticate

**Comment by cj848** [comment]
oh. Thank you. I checked. It works fine. please release it!

**Comment by mateuszkwiecinski** [comment]
Fixed in `v0.4.1`
